### ISSUE TITLE
Documentation - Async Path Operation Function 

### ISSUE THREAD

**Issue by HarshKhatore** [tiangolo/fastapi#3070]
### Description

* Go to https://fastapi.tiangolo.com/async/#path-operation-functions
* Shouldn't the places of 'def' and 'async def' be interchanged? From the doc - When you declare a path operation function with normal def instead of async def, it is run in an external threadpool that is then awaited, instead of being called directly (as it would block the server). - to me, this defines the behaviour of an async def rather than a def.

**Comment by Kludex** [comment]
No. `await` makes "you" able to suspend the execution of that coroutine (`async`).

On functions (`def`), we use a threadpool to achieve the same goal, but in a more costly way in comparison to the "`async/await` way". 

References:
- https://docs.python.org/3/glossary.html#term-coroutine
- https://www.python.org/dev/peps/pep-0492/#await-expression
- https://www.python.org/dev/peps/pep-0492/ 
- https://docs.python.org/3/library/asyncio.html

**Comment by HarshKhatore** [comment]
Ah, okay, this makes it clear. Thanks!

**Comment by tiangolo** [comment]
Thanks for the help here @Kludex ! 👏  🙇 

Thanks for reporting back and closing the issue @HarshKhatore  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
